docs/localized/zh-CN/README.md

```diff
@@ -32,7 +32,7 @@ TensorFlow），让游戏开发者和业余爱好者能够轻松地
 **除了安装和使用说明外，如需更多信息，
 请参阅我们的[文档主页](docs/Readme.md)。**如果您使用的
 是 v0.3 之前的 ML-Agents 版本，强烈建议您参考
-我们的[关于迁移到 v0.3 的指南](/docs/Migrating.md)。
+我们的[关于迁移到 v0.3 的指南](../../../docs/Migrating.md)。
```

Comment:
@ervteng @unityjeffrey Let me know what you think about this - to "officially" get the link checker running, we need to have the links from the localized docs point somewhere, because they doc they do point to hasn't been localized. I figure this is better since it's at least explicit in the markdown that it's linking somewhere outside the localization.

Alternative is just whitelist the localized docs and skip the checks on then.